### ISSUE TITLE
The problem of regression to the pixel value of the picture

### ISSUE THREAD

**Issue by Doctor-James** [YyzHarry/imbalanced-regression#36]
I have the honor to read your paper, which is very solid. Now I am doing a task. label is a 256 * 256 picture, which needs to predict each pixel value of it (unbalanced continuous value between 0 and 1). Now I want to apply your LDS strategy, but my task is slightly different from yours. Although my data set is only 1W pieces, if each pixel is regarded as a label, the label will be very large（256 * 256 *10000）. Do you think your method is still applicable?
I did an experiment where I calculated the value distribution for each label(256 * 256 images) separately and then applied the LDS strategy. It didn't work out too well.

**Comment by YyzHarry** [comment]
Thanks for your interest. Your problem seems to be pretty similar to the setting of the [NYUD2-DIR](https://github.com/YyzHarry/imbalanced-regression/tree/main/nyud2-dir) task we provided in the codebase, where we would like to predict the per-pixel dense depth map. As reported from Table 11 in our paper, FDS & LDS+FDS seem to work better for this dataset. We recommend you also try FDS and the combination of the two.

In the meantime, you might also want to adjust the hyper-parameters (e.g., `kernel_size`, `sigma`) to better fit your task. In Appendix E.3 of our [paper](https://arxiv.org/pdf/2102.09554.pdf), we studied some choices of kernel_size and sigma, and it might give you some sense on what values are good for different tasks.

**Comment by Doctor-James** [comment]
Thank you for your reply，about NYUD2-DIR task，I have one question
I found that you set another train_fds_loader in this task, which is not in other tasks. train_fds_loader and train_loader are the same in other tasks，Is there anything special about this task？
Thanks！

**Comment by YyzHarry** [comment]
For the original NYUD2 dataset, the standard data augmentation in the field contains many customized [operations](https://github.com/YyzHarry/imbalanced-regression/blob/main/nyud2-dir/loaddata.py#L96); to remove the effects of these augmentations for FDS to calculate the feature statistics, we thereby used two loaders separately.
